src/main/webapp/components/ErrorBoundary.jsx

```diff
@@ -31,20 +32,30 @@ class ErrorBoundary extends React.Component {
         if (this.state.error) {
             // Error path
             return (
-                <>
-                    <Heading level={2}>
-                        {_('Something went wrong!')}
-                        {this.state.errorCode ? ` ERROR_CODE: ${this.state.errorCode}` : null}
-                    </Heading>
-                    {this.state.errorCode ? (
-                        <Message type="info">{errorCodes[this.state.errorCode]}</Message>
-                    ) : null}
-                    <details style={{ whiteSpace: 'pre-wrap' }}>
-                        {this.state.error?.toString()}
-                        <br />
-                        {this.state.errorInfo?.componentStack}
-                    </details>
-                </>
+                <div align="center" style={{ marginTop : "10%" }}>
+                <Card style={{ boxShadow : "10px 10px 5px #aaaaaa" }} >
+                    <Card.Header>
+                        <Heading style={{ textAlign:"center" }} level={2}>
+                            <WarningIcon style={{ fontSize: "120px", color: "#ff9900" }} /><br/><br/>
+                            {this.state.errorCode === _('ERR0001') ? _('Failed to load Inputs Page') : _('Something went wrong!')}
```

Comment:
`errorCode` itself doesn't need to be wrapped in `_("abc")`. We wrap strings which are language words and can be translated automatically to other language.

Reply:
Made the necessary change.